.circleci/config.yml

```diff
@@ -13,6 +13,7 @@ orbs:
   kubernetes: circleci/kubernetes@0.11.2
   mem: circleci/rememborb@0.0.1
   snyk: snyk/snyk@0.0.12
+  veracode: orb-01scan/sast-orb@0.0.9
```

Comment:
weird import path eh?

Reply:
Yeah. Its support for CircleCI started recently, so it's a spike I think